File: ios/Kwift/com/lightningkite/kwift/observables/Event.actual.swift
import RxSwift
import RxRelay

//--- Observer<Element>.invokeAll(Element)
extension ObserverType {
     public func invokeAll(value: Element) -> Void {
        return onNext(value)
    }
     public func invokeAll(_ value: Element) -> Void {
        return invokeAll(value: value)
    }
}

File: android/src/main/java/com/lightningkite/khrysalis/observables/Event.actual.kt
package com.lightningkite.khrysalis.observables

import com.lightningkite.khrysalis.AnyObject
import com.lightningkite.khrysalis.weak
import io.reactivex.Observable
import io.reactivex.Observer
import io.reactivex.disposables.Disposable
import io.reactivex.rxkotlin.subscribeBy
import io.reactivex.subjects.PublishSubject
import io.reactivex.subjects.Subject
import java.util.*

@Deprecated("Use RX directly instead", ReplaceWith("onNext(value)"))
fun <Element> Observer<Element>.invokeAll(value: Element) = onNext(value)
